gpt_engineer/benchmark/benchmarks/apps/load.py

```diff
@@ -0,0 +1,76 @@
+"""
+Module for loading APPS evaluation tasks.
+
+This module provides functionality to load tasks for evaluating GPT-based models
+on smaller, more focused tasks. It defines a set of tasks with predefined prompts
+and assertions to benchmark the performance of AI models.
+
+Functions
+---------
+load_apps : function
+    Loads the APPS benchmark, which consists of a series coding problems.
+"""
+from typing import Union
+
+from gpt_engineer.benchmark.benchmarks.apps.problem import Problem
+from gpt_engineer.benchmark.benchmarks.apps.problems import PROBLEM_IDS
+from gpt_engineer.benchmark.types import Benchmark, Task
+from gpt_engineer.core.files_dict import FilesDict
+from datasets import load_dataset, load_from_disk, Dataset, DatasetDict
+
+DATASET_PATH = "gpt_engineer/benchmark/benchmarks/apps/dataset"
```

Comment:
Make this path absolute, so that it doesn't depend on the working directory.

```suggestion
from pathlib import Path
DATASET_PATH = Path(__file__).parent / "dataset"
```